src/Homeworks/Calculator/Calculator/Calculator.cs

```diff
@@ -0,0 +1,63 @@
+namespace CalculatorApp
+{

```

Comment:
```suggestion
namespace CalculatorApp;
```
В моде уже file-scoped неймспейсы. В других файлах аналогично

Reply:
В тестах не исправлено